bigchaindb/tendermint/utils.py

```diff
@@ -15,6 +15,12 @@ def decode_transaction(raw):
     return json.loads(raw.decode('utf8'))
 
 
+def decode_txn_base64(value):
```

Comment:
In general, we try to keep the names of functions and variable explicit, so what about a simple `decode_transaction_base64`?

Moreover, would it make sense to also do a `json.loads` inside the function?